bin/install

```diff
@@ -52,8 +53,8 @@ install_canon_version() {
 
       # shellcheck disable=SC2086
       ./configure $configure_options || exit 1
-      make
-      make install
+      make $make_flags || exit 1
+      make $make_flags install || exit 1
```

Comment:
```suggestion
      make "$make_flags" || exit 1
      make "$make_flags" install || exit 1
```

Althought this would actually cause a problem if `make_flags` contained more than one flag, as the make command would see all flags as one argument instead of individual arguments. I am fine with merging this code as it is.